recipe/meta.yaml

```
@@ -1,40 +1,42 @@
 {% set name = "idna" %}
-{% set version = "3.7" %}
+{% set version = "3.11" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 028ff3aadf0609c1fd278d8ea3089299412a7a8b9bd005dd08b9f8285bcb5cfc
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 795dafcc9c04ed0c1fb032c2aa73654d8e8c5023a7df64a53f39190ada629902
 
 build:
   number: 0
-  skip: True  # [py<36]
+  skip: true  # [py<38]
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed --no-cache-dir -vv
 
 requirements:
   host:
-    - flit-core >=3.2,<4
     - python
     - pip
-    - wheel
-    # setuptools is needed for legacy users, see https://github.com/kjd/idna/commit/9234d29b38e56e863db03af5980819c031221725
-    - setuptools
+    - flit-core >=3.11,<4
   run:
     - python
```

```suggestion
    - python
  run_constrained:
    - ruff >=0.6.2
    - mypy >= 1.11.2
```

https://github.com/kjd/idna/blob/v3.11/pyproject.toml#L47

---

I dont think it is needed optional run_constrained since it is type checker and linter, it is more for dev